gems/pending/VolumeManager/MiqLvm.rb

```diff
@@ -295,6 +302,436 @@ def parseString
   end
 end # class Lvm2MdParser
 
+# move to BinaryStruct
```

Comment:
@movitto can you pull this out and make a PR on https://github.com/ManageIQ/binary_struct /cc @Fryguy